source/support.tex

```diff
@@ -712,6 +712,8 @@
 #define @\defnlibxname{cpp_lib_to_underlying}@                     202102L // also in \libheader{utility}
 #define @\defnlibxname{cpp_lib_transformation_trait_aliases}@      201304L // also in \libheader{type_traits}
 #define @\defnlibxname{cpp_lib_transparent_operators}@             201510L // also in \libheader{memory}, \libheader{functional}
+#define @\defnlibxname{cpp_lib_tuple_like}@                        202207L // also in \libheader{utility}, \libheader{tuple}, \libheader{map},
```

Comment:
https://github.com/cplusplus/draft/pull/5694#discussion_r933822832 is still missing as suggested.

Reply:
done